### PR TITLE
Fix disabled args appearing in documentation.

### LINKED ISSUE #12581: brew tap --full <repo> failing
### `brew config` output

```shell
HOMEBREW_VERSION: 3.3.8
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3f0b412951996a675b8a48037e9a978f0ccd8363
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 2d78cf0cf152ee08c765c3a6c483c6a44a85577e
Core tap last commit: 43 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 13.0.0 build 1300
Git: 2.30.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.6.1-x86_64
CLT: 13.0.0.0.1.1630607135
Xcode: N/A
```

### `brew doctor` output

```shell
Your system is ready to brew.
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

trying to use `brew tap --full` or `brew tap --shallow`. The Chef.io built-in homebrew resources are failing because these options aren't working. 

### What happened (include all command output)?

Calling `brew tap --full` is disabled! There is no replacement.
Calling `brew tap --shallow` is disabled! There is no replacement.

### What did you expect to happen?

The brew man page still indicates support for these options.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
executing `brew tap --full <repo>` will reproduce this error.
```

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Closes #12581.

### CODE REVIEW

File: Library/Homebrew/dev-cmd/pr-automerge.rb [hunk 1]
```diff
@@ -28,7 +28,8 @@ def pr_automerge_args
              description: "Run `brew pr-publish` on matching pull requests."
       switch "--autosquash",
              description: "Instruct `brew pr-publish` to automatically reformat and reword commits "\
-                          "in the pull request to our preferred format."
+                          "in the pull request to our preferred format.",
+             replacement: "`--no-autosquash` to out out"
```
Comment (Rylan12) [inline Library/Homebrew/dev-cmd/pr-automerge.rb:32]:
```suggestion
             replacement: "`--no-autosquash` to opt out"
```

File: Library/Homebrew/dev-cmd/pr-publish.rb [hunk 2]
```diff
@@ -18,7 +18,8 @@ def pr_publish_args
       EOS
       switch "--autosquash",
              description: "If supported on the target tap, automatically reformat and reword commits "\
-                          "in the pull request to our preferred format."
+                          "in the pull request to our preferred format.",
+             replacement: "`--no-autosquash` to out out"
```
Comment (Rylan12) [inline Library/Homebrew/dev-cmd/pr-publish.rb:22]:
```suggestion
             replacement: "`--no-autosquash` to opt out"
```

**Review by Rylan12 — COMMENTED** [review]
👍 aside from two typos

**Review by Rylan12 — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Great work here, thanks @Bo98!

**Pull request merged**